src/nx.ts

```diff
@@ -26,7 +26,15 @@ const processWorkspaceJson: PostProcessWorkspaceFn = (filterFn) => (out) => {
   try {
     const workspace = JSON.parse(out);
     return Object.entries<NxProject>(workspace.projects)
-      .filter(filterFn)
+      .filter((el, ind, arr) => {
+        function filterFn(el, ind, arr) {
+          if (el == null || ind == null || arr == null) {
+            return false;
+          }
+          return true;
+        }
+        return filterFn(el, ind, arr);
+      })
```

Comment:
You can condense this down by using an anonymous function, like this:
```suggestion
      .filter((elm, idx, arr) => elm == null || idx == null || arr == null)
```
Otherwise looks good! Thank you for contributing :)